packages/core/tests/js/common/expression-from-string/output.ts

```diff
@@ -0,0 +1 @@
+export default defineConfig({ path: 'some/string/as/path.js', valid: true });
```

Comment:
To be exact, this would be my expectation because [`guessQuoteStyle`](https://github.com/sveltejs/cli/blob/e9485a2c8e6df490291490d91059a2540ab9653a/packages/core/tooling/index.ts#L211) should not be able to find any quotes and thus return `undefined`. Therefore the `esrap` default quote style 'single' should apply: https://github.com/sveltejs/esrap?tab=readme-ov-file#options